### PR TITLE
Fix NDEV-13906

### PR DESCRIPTION
**Pull request by mrutyunjaya-nirmata**
Remove the patch from baseline kustomization

### CODE REVIEW

**Comment by mrutyunjaya-nirmata** [comment]
@atulatnirmata @chipzoller Can you please review it?

File: pod-security/baseline/kustomization.yaml [hunk 1]
```diff
@@ -10,12 +10,4 @@ resources:
...
-
-patches:
-  - patch: |-
-      - op: replace
-        path: /spec/validationFailureAction
-        value: enforce
-    target:
-      kind: ClusterPolicy
\ No newline at end of file
+  - restrict-sysctls/restrict-sysctls.yaml
```
Comment (atulatnirmata) [inline pod-security/baseline/kustomization.yaml]: Please add a new line

**Review by atulatnirmata — APPROVED** [review]
Please incorporate the minor comment
Read this https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=If%20content%20is%20added%20to,you%20actually%20wanted%20to%20see. on why new line should be added to any file

**Comment by mrutyunjaya-nirmata** [comment]
@nsagark Can you please review PR?

**Review by chipzoller — APPROVED** [review]

**Comment by atulatnirmata** [comment]
@anubhav888, @nsagark - can one of you merge this?

**Review by atulatnirmata — APPROVED** [review]

**Pull request merged**